stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -33,11 +33,10 @@ endif()
 # Frustratingly, in many cases this isn't necessary because the
 # sequence is inlined, but we have some code that's just subtle
 # enough to turn into runtime calls.
-if(SWIFT_HOST_VARIANT STREQUAL "Linux")
-  if(SWIFT_HOST_VARIANT_ARCH MATCHES "armv6|armv7|i686")
-    list(APPEND swift_concurrency_link_libraries
-      atomic)
-  endif()
+if(SWIFT_HOST_VARIANT STREQUAL "linux" OR
```

Comment:
Was the change from `Linux` to `linux` on purpose?

Reply:
Yes, `SWIFT_HOST_VARIANT` seems to generally be lowercase.  That's the primary thing this fixes.

Reply:
As far as I can tell, the code I copy-and-pasted from is inoperative.

Reply:
A shorter way to write this would be `SWIFT_HOST_VARIANT MATCHES "linux|android"`, thanks for adding Android.

Reply:
Are the legal values for `SWIFT_HOST_VARIANT` documented somewhere?

Reply:
[They are defined here](https://github.com/apple/swift/blob/2464989c4244f91dfdc75236611c010f6bb00813/CMakeLists.txt#L728), I count eight possible hosts.